src/rc-conf-generator/rc.conf

```diff
@@ -0,0 +1,62 @@
+#
+# /etc/rc.conf - Main Configuration for Modern Linux
+#
+
+# -----------------------------------------------------------------------
+# LOCALIZATION
+# -----------------------------------------------------------------------
+#
+# LOCALE: available languages can be listed with the 'locale -a' command
+# TIMEZONE: timezones are found in /usr/share/zoneinfo
+# KEYMAP: keymaps are found in /usr/share/kbd/keymaps
+#
+LOCALE="en_US.UTF-8"
+TIMEZONE="Europe/Berlin"
+KEYMAP="us"
+
+# -----------------------------------------------------------------------
+# NETWORKING
+# -----------------------------------------------------------------------
+#
+# HOSTNAME: Hostname of machine. Should also be put in /etc/hosts
+#
+HOSTNAME="myhost"
+
+# Use 'ip addr' or 'ls /sys/class/net/' to see all available interfaces.
+#
+# Wired network setup
+#   - interface: name of device (required)
+#   - address: IP address (leave blank for DHCP)
+#   - netmask: subnet mask (ignored for DHCP) (optional, defaults to 255.255.255.0)
+#   - broadcast: broadcast address (ignored for DHCP) (optional)
+#   - gateway: default route (ignored for DHCP)
+#
+# Static IP example
+# interface=eth0
+# address=192.168.0.2
+# netmask=255.255.255.0
+# broadcast=192.168.0.255
+# gateway=192.168.0.1
+#
+# DHCP example
+# interface=eth0
+# address=
+# netmask=
+# gateway=
+
+interface=
+address=
+netmask=
+broadcast=
+gateway=
+
+
+# -----------------------------------------------------------------------
+# DAEMONS
+# -----------------------------------------------------------------------
+#
+# Daemons to start at boot-up (in this order)
+#   - prefix a daemon with a ! to disable it
+#   - prefix a daemon with a @ to start it up in the background
+#
+DAEMONS=(nginx @sshd !lvm2-monitor)
```

Comment:
One of the most valuable features ever observed in this tracker. Why ever work with elaborate and convoluted configs when you had everything presented in a single file. Units? Drop-ins? Never heard of them.

Small nit though: arrays are bashism. To give people 100% original flavor let's embrace posix sh